lib/routes/openrice/namespace.ts

```diff
@@ -5,5 +5,5 @@ export const namespace: Namespace = {
     url: 'www.openrice.com',
     categories: ['shopping'],
     description: '美食網站Openrice相关資訊',
-    lang: 'zh-HK',
+    lang: 'zh',
```

Comment:
```suggestion
    lang: 'zh-HK',
```
`zh` is not listed in https://github.com/DIYgod/RSSHub/blob/fd5a888f185d97c7679dfb724a63b31b6de51638/lib/types.ts#L93